src/components/common/PageLayout/styles.module.css

```diff
@@ -1,18 +1,11 @@
-.header {
-  position: fixed;
-  left: 0;
-  top: 0;
-  width: 100%;
-  z-index: 1201;
-}
```

Comment:
Why move this? PageLayout is responsible for the positions of the elements in it. Header is only responsible for what's inside the header.